diff --git a/riscv64/core_portme.h b/riscv64/core_portme.h
index 4e28afd..1e7708e 100755
--- a/riscv64/core_portme.h
+++ b/riscv64/core_portme.h
@@ -71,7 +71,7 @@ typedef clock_t CORE_TICKS;
      Note these need to match the size of the clock output and the xLen the processor supports
  */
 typedef unsigned long int size_t;
-typedef unsigned long int clock_t;
+typedef long int clock_t;
 typedef clock_t CORE_TICKS;
 #endif
 
