diff --git a/builtins/common.c b/builtins/common.c
index 00be24e..3fb177f 100644
--- a/builtins/common.c
+++ b/builtins/common.c
@@ -84,21 +84,6 @@ sh_builtin_func_t *this_shell_builtin = (sh_builtin_func_t *)NULL;
    instead of shell control structures, and it won't ever exit the
    shell. */
 
-static void
-builtin_error_prolog ()
-{
-  char *name;
-
-  name = get_name_for_error ();
-  fprintf (stderr, "%s: ", name);
-
-  if (interactive_shell == 0)
-    fprintf (stderr, _("line %d: "), executing_line_number ());
-
-  if (this_command_name && *this_command_name)
-    fprintf (stderr, "%s: ", this_command_name);
-}
-
 void
 #if defined (PREFER_STDARG)
 builtin_error (const char *format, ...)
@@ -110,8 +95,6 @@ builtin_error (format, va_alist)
 {
   va_list args;
 
-  builtin_error_prolog ();
-
   SH_VA_START (args, format);
 
   vfprintf (stderr, format, args);
@@ -130,7 +113,6 @@ builtin_warning (format, va_alist)
 {
   va_list args;
 
-  builtin_error_prolog ();
   fprintf (stderr, _("warning: "));
 
   SH_VA_START (args, format);
