xinitrc.cpp: use Fluxbox instead of TWM

Signed-off-by: Grzegorz Latosinski <glatosinski@antmicro.com>

--- xinit-1.4.1.orig/xinitrc.cpp	2019-03-03 20:58:11.000000000 +0100
+++ xinit-1.4.1/xinitrc.cpp	2019-07-17 15:13:18.524034334 +0200
@@ -48,8 +48,4 @@ if [ -d XINITDIR/xinitrc.d ] ; then
 	unset f
 fi
 
-TWM &
-XCLOCK -geometry 50x50-1+1 &
-XTERM -geometry 80x50+494+51 &
-XTERM -geometry 80x20+494-0 &
-exec XTERM -geometry 80x66+0+0 -name login
+( fluxbox -no-toolbar & )
